Gruntfile.js

```diff
@@ -167,6 +167,13 @@ module.exports = function(grunt) {
                 expand: true,
                 flatten: true
             },
+            bootOverCss: {
```

Comment:
```suggestion
            bootstrapOverrideCss: {
```
Recommend to use full name cause short isn't really understandable